Oraimunka24015004/src/AlgorithmicRecap.java

```diff
@@ -0,0 +1,172 @@
+import java.math.BigInteger;
+import java.util.*;
+
+public class AlgorithmicRecap {
+    public String taskOneReverts(String text) {
+        String result = "";
+        for (int i = text.length(); i >= 1; i--) {
+            char charcter = text.charAt(i - 1);
+            result = result + charcter;
```

Comment:
please use `StringBuilder` - creating Strings every time is really costly (remember that Strings are immutable by nature)